index.js

```diff
@@ -1,6 +1,7 @@
 // express setup
 const express = require('express')
 const app = express()
+const bcrypt = require('bcrypt')
 
 const crypto = require('crypto');
```

Comment:
Once we pick one can remove the other.

Reply:
Actually, not true anymore. Even if we go with bcrypt, crypto can be used for authenticating later.